sources/packages/api/src-sql/SFASPartTimeApplications/Create-part-time-sfas-application-records.sql

```diff
@@ -0,0 +1,37 @@
+-- CREATE TABLE sfas_part_time_applications
+CREATE TABLE IF NOT EXISTS sims.sfas_part_time_applications (
+  id VARCHAR(10) PRIMARY KEY,
+  individual_id INT NOT NULL,
+  start_date DATE,
+  end_date DATE,
+  csgp_award NUMERIC(10),
+  sbsd_award NUMERIC(10),
```

Comment:
If we are saving the awards it would be better to have decimals in it while defining the table like `NUMERIC(12, 2)`, Similar to the sfas_applications table

Reply:
It seems that these number does not have decimals, as per below documenttion.
![image](https://user-images.githubusercontent.com/61259237/150197802-fee1a2a1-5eaf-49e0-9a7a-e743d9aacab7.png)
@abschwenker, can you confirm it?

Reply:
yes, as @andrewsignori-aot said these awards doesn't have decimals